src/Integration/LocalServices/RuleSetInspector.cs

```diff
@@ -48,15 +48,65 @@ public RuleSetInspector(IServiceProvider serviceProvider, params string[] userRu
             ruleSetSearchDirectories.Add(GetStaticAnalysisToolsDirectory());
         }
 
+        /// <summary>
+        /// <see cref="IRuleSetInspector.FindConflictingRules(string, string)"/>
+        /// </summary>
+        public RuleSet FixConflictingRules(string baselineRuleSetPath, string targetRuleSetPath)
```

Comment:
I considered adding the RuleConflictInfo as an argument but since the find and the fix could happen with a delay (in which the user can update the ruleset) we would have to recalculate the conflicts anyway to be able to fix them with 100% certainty.